core/meterer/onchain_state.go

```diff
@@ -14,9 +14,7 @@ import (
 // OnchainPaymentState is an interface for getting information about the current chain state for payments.
 type OnchainPayment interface {
 	RefreshOnchainPaymentState(ctx context.Context, tx *eth.Transactor) error
-	GetActiveReservations(ctx context.Context) (map[string]core.ActiveReservation, error)
```

Comment:
Do we no longer need these getters? How is the internal state being accessed?

Reply:
We never used these getters, so I decided to remove them. 

The internal state gets accessed in other OnchainPayment methods for either the [length](https://github.com/Layr-Labs/eigenda/blob/48ac72718d68beab5af6d0a6a078872b0e822b61/core/meterer/onchain_state.go#L59) of the map (with locks) or account ID specific inner [struct](https://github.com/Layr-Labs/eigenda/blob/48ac72718d68beab5af6d0a6a078872b0e822b61/core/meterer/onchain_state.go#L89) (no lock). 

I could add a lock around accessing the internal state?

Reply:
Got it. These reads look fine to me!